core/src/main/java/org/infinispan/util/logging/events/Messages.java

```diff
@@ -71,4 +71,10 @@
 
    @Message(value = "After merge (or coordinator change), the coordinator failed to recover cluster. Cluster members are %s.")
    String clusterRecoveryFailed(Collection<Address> members);
+
+   @Message(value = "Site '%s' is online.", id = 100004)
```

Comment:
because the messages start with id 100000.
the messages with id 300+ was moved from the ```Log``` as it is in the comment:
```
// Messages between 312 and 320 have been moved to the org.infinispan.util.logging.events.Messages class
```
the ```eventLog*``` has no id because it is used to build the final message (from what I understand)
finally, I have no idea why the last message has no id.